css_tasks/task11/scss/_home.scss

```diff
@@ -0,0 +1,192 @@
+.home__background {
+  position: relative;
+  :after {
```

Comment:
wouldn't it better to add after to specific element, but not to all children of home background

Reply:
this ::after adds second background  to the whole section ( gradient color). I can't add :after to .home-background__image, because <img/> is a void element and you can't just do that. I don't understand what you mean. I need second background to have the same width and height as the image so where else I should add ::after? :)

Reply:
but in result html it looks as added to .home-background__image

i meant that it is better that after element was clearly assign to some element on page